jwst/assign_wcs/util.py

```diff
@@ -101,7 +102,8 @@ def _reproject(x, y):
 
 
 def compute_scale(wcs: WCS, fiducial: Union[tuple, np.ndarray],
-                  disp_axis: int = None, pscale_ratio: float = None) -> float:
+                  disp_axis: int = None, pscale_ratio: float = None,
+                  wavelength: float=None) -> float:
```

Comment:
code style might complain about the lack of whitespace around the "=" here?

Reply:
yeah, I'll fix those